### PR TITLE
refactor: renamed class AddChildrenAction to AddChildren

### LINKED ISSUE #956: Renamed class AddChildrenAction to AddChildren
Please provide all the information requested. Issues that do not follow this format are likely to stall.

## Description
On the IOS and web platforms they are following the contract under the name AddChildren, but on the Android and Backend platforms they are under the name AddChildrenAction

## Steps To Reproduce
Provide a detailed list of steps that reproduce the issue.

1.Follow the backend project link:
https://github.com/ZupIT/beagle/blob/master/backend/widgets/src/main/kotlin/br/com/zup/beagle/widget/action/AddChildrenAction.kt
2..Follow the android project link:
https://github.com/ZupIT/beagle/blob/master/android/beagle/src/main/java/br/com/zup/beagle/android/action/AddChildrenAction.kt

## Expected Results
Result that all platforms follow the contract defined as AddChildren

## Code example, screenshot, or link to a repository:
Please provide a link to a repository on GitHub, or provide a minimal code example that reproduces the problem.
You may provide a screenshot of the application if you think it is relevant to your bug report.
Here are some tips for providing a minimal example: https://stackoverflow.com/help/mcve

### PR DESCRIPTION
**Pull request by luisoliveirazup**
### Related Issues
closes #956 

### Description and Example
On the IOS and web platforms they are following the contract under the name AddChildren, but on the Android and Backend platforms they are under the name AddChildrenAction

expected: Result that all platforms follow the contract defined as AddChildren

BREAKING CHANGE:  Action renamed   AddChildrenAction to AddChildren.

### Checklist

Please, check if these important points are met using `[x]`:

- [x] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [x] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [x] I am willing to follow-up on review comments in a timely manner.

[PR Guide]: https://github.com/ZupIT/beagle/blob/master/doc/contributing/pull_requests.md

### CODE REVIEW

File: android/beagle/src/test/java/br/com/zup/beagle/android/data/serializer/BeagleMoshiTest.kt [hunk 1]
```diff
@@ -495,6 +496,19 @@ class BeagleMoshiTest : BaseTest() {
         assertTrue(actual is Condition)
     }
 
+    @Test
+    fun `make should return moshi to deserialize a AddChildren` (){
```
Comment (uziasferreirazup) [inline android/beagle/src/test/java/br/com/zup/beagle/android/data/serializer/BeagleMoshiTest.kt:500]: please follow the new pattern name:
example: ``GIVEN response data is an integer WHEN toResponse is called THEN it should return an Int object``()

Comment (uziasferreirazup) [inline android/beagle/src/test/java/br/com/zup/beagle/android/data/serializer/BeagleMoshiTest.kt:500]: add the test to serialization and deserialization, you can look at the tests to know how to do:
 fun `make should return moshi to deserialize a Container`
 fun `make should return moshi to serialize a Container`

**Review by uziasferreirazup — CHANGES_REQUESTED** [review]

File: android/beagle/src/test/java/br/com/zup/beagle/android/data/serializer/BeagleMoshiTest.kt (hunk 1 above)
Comment (luisoliveirazup) [reply android/beagle/src/test/java/br/com/zup/beagle/android/data/serializer/BeagleMoshiTest.kt]: change done!

Comment (luisoliveirazup) [reply android/beagle/src/test/java/br/com/zup/beagle/android/data/serializer/BeagleMoshiTest.kt]: change done!

**Review by uziasferreirazup — APPROVED** [review]

**Review by viniciusguardieirozup — APPROVED** [review]

**Pull request merged**